modules/domUtils.js

```diff
@@ -0,0 +1,27 @@
+// import { showContent } from './domUtils.js';
```

Comment:
- [ ] Please do not leave commented-out code in your codebase, it makes your project look messy.